### PR TITLE
Add tests for log configuration

### PR DESCRIPTION
**Pull request by tempoz**
---

**Version bump**: None

### CODE REVIEW

**Comment by tempoz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3015** <a href="https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/3015" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/buildbuddy-io/buildbuddy/3015?utm_source=stack-comment).

File: server/util/log/log_test.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	assert.Equal(t, zl.LevelFieldName, "level")
+	assert.Equal(t, zl.TimestampFieldName, "time")
+}
+
+func TestConfigureEnableStructuredLogging(t *testing.T) {
+	flags.Set(t, "app.enable_structured_logging", true)
+	log.Configure()
+	assert.Equal(t, zl.LevelFieldName, "severity")
+	assert.Equal(t, zl.TimestampFieldName, "timestamp")
+}
```
Comment (bduffany) [inline server/util/log/log_test.go:62]: nit: it's not obvious to me what these assertions are doing, mind adding a comment?

**Review by bduffany — APPROVED** [review]
Thanks for adding these!

File: server/util/log/log_test.go (hunk 1 above)
Comment (tempoz) [reply server/util/log/log_test.go]: Done!

**Pull request merged**